test/cppsuite/test_harness/workload_generator.h

```diff
@@ -99,13 +103,22 @@ class workload_generator : public component {
         /* Open a cursor on each collection and use the configuration to insert key/value pairs. */
         testutil_check(_config->get_int(KEY_COUNT, key_count));
         testutil_check(_config->get_int(VALUE_SIZE, value_size));
-        testutil_assert(value_size >= 0);
+        testutil_assert(value_size > 0);
+        testutil_check(_config->get_int(KEY_SIZE, key_size));
+        testutil_assert(key_size > 0);
+        /* Keys must be unique. */
+        testutil_assert(key_count <= pow(10, key_size));
+
         for (const auto &collection_name : _collection_names) {
```

Comment:
Can we store the collection names in the database instead of duplicating them to `_collection_names` and maybe add a function to the Database which is something like "iterate_collections()"?